### PR TITLE
Added pdf factory, controller trait und local_asset twig extension

### PR DESCRIPTION
**Pull request by alexander-schranz**

### CODE REVIEW

File: Pdf/PdfFactory.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    /**
+     * PdfFactory constructor.
+     *
+     * @param EngineInterface $templating
```
Comment (wachterjohannes) [inline Pdf/PdfFactory.php]: remove this because its already outdated

Comment (alexander-schranz) [reply Pdf/PdfFactory.php]: depends of if we release a major version or not?

Comment (wachterjohannes) [reply Pdf/PdfFactory.php]: hmm not really :) removing the phpdoc is not a BC break and as I said at least you would have to add the `$requestStack` to the phpdoc

Comment (alexander-schranz) [reply Pdf/PdfFactory.php]: ah sorry thought we are in the PdfManager and you want to remove the whole PdfManager service

**Pull request merged**